### PR TITLE
Surgery patients scream if they're not anaesthetized

### PR DESCRIPTION
**Pull request by DamianX**
Doesn't do pain damage, doesn't cause surgery to fail, no side-effects whatsoever.
It's just louder.

### CODE REVIEW

**Comment by brndd** [comment]
does sleeping count as anesthesia?

**Comment by DamianX** [comment]
yes

**Comment by SonixApache** [comment]
does alcohol / painkillers factor into this because it should

**Comment by DamianX** [comment]
yes

File: code/modules/organs/pain.dm [hunk 1]
```diff
@@ -45,7 +45,7 @@ mob/living/carbon/proc/pain(var/partname, var/amount, var/force, var/burning = 0
 
 // message is the custom message to be displayed
 // flash_strength is 0 for weak pain flash, 1 for strong pain flash
-mob/living/carbon/human/proc/custom_pain(var/message, var/flash_strength)
+mob/living/carbon/human/proc/custom_pain(var/message, var/flash_strength, var/scream = FALSE)
 	if(stat >= 1)
```
Comment (PaulRitter) [inline code/modules/organs/pain.dm:49]: `stat > CONSCIOUS` would be better here imo

we got the defines, why not use em

**Review by PaulRitter — COMMENTED** [review]
so turns out my review never submitted

File: code/modules/organs/pain.dm (hunk 1 above)
Comment (DamianX) [reply code/modules/organs/pain.dm:49]: mostly because I didn't write this

**Pull request merged**